src/VStore/Kafka/EventReceiver.cs

```
@@ -41,6 +40,12 @@ public EventReceiver(ILogger logger, string brokerEndpoints, string groupId)
                             }
                     }
                 };
+
+            if (!string.IsNullOrEmpty(groupId))
```

I think we could throw `InvalidOperationException` in case of calling `EventReceiver.CommitAsync()` without consumer group